11ty/definitions/derpy.md

```diff
@@ -0,0 +1,29 @@
+---
+title: derpy
+slug: derpy
+flag:
+  text: 'Ableist Term'
+  level: avoid
+defined: true
+speech: adjective
+alt_words:
+  - ridiculous
+  - clumsy
+  - foolish
+  - misshapen
+  - malformed
+---
+
+clumsy, foolish, unattractive, or otherwise less intelligent person; 
+
+## Issues
+
+Derpy is very commonly used as an adjective to describe someone's clumsy or foolish behavior, often that of the speaker themselves. It is also used to describe someone else's physical features as unattractive or akin to someone with intellectual and developmental disabilities. In imagery derpy is often signified by crossed eyes, a stuck out tongue, or facial deformities. 
+
+## Impact
+
+By using ableist language, we are perpetuating violence against people who experience mental or physical disabilities. Using this language perpetuates those systems and language of harm, regardless of our intent.
```

Comment:
I think adding a bit more context here would be helpful! The definition for `r-word` has excellent context on intellectual and development disabilities!

Reply:
I'm taking a look around, clicking some buttons and making some changes. I'll take a look at that definition to incorporate more context.